src/nunit.analyzers.tests/ClassicModelAssertUsage/IsAssignableFromClassicModelAssertUsageCodeFixTests.cs

```diff
@@ -0,0 +1,440 @@
+            var actual = 42;
+
+            ↓ClassicAssert.IsAssignableFrom<int>(actual, ""message-id: {0}"", Guid.NewGuid());");
+            var fixedCode = TestUtility.WrapInTestMethod(@"
+            var actual = 42;
+
+            Assert.That(actual, Is.AssignableFrom<int>(), $""message-id: {Guid.NewGuid()}"");");
+            RoslynAssert.CodeFix(analyzer, fix, expectedDiagnostic, code, fixedCode, fixTitle: ClassicModelAssertUsageCodeFix.TransformToConstraintModelDescription);
+        }
+
+        [Test]
+        public void VerifyIsAssignableFromGenericFixWithMessageAndTwoArgumentsForParams()
+        {
+            var code = TestUtility.WrapInTestMethod(@"
+            var actual = 42;
+
+            ↓ClassicAssert.IsAssignableFrom<int>(actual, ""{0}, {1}"", ""first"", ""second"");");
+            var fixedCode = TestUtility.WrapInTestMethod(@"
+            var actual = 42;
+
+            Assert.That(actual, Is.AssignableFrom<int>(), $""{""first""}, {""second""}"");");
+            RoslynAssert.CodeFix(analyzer, fix, expectedDiagnostic, code, fixedCode, fixTitle: ClassicModelAssertUsageCodeFix.TransformToConstraintModelDescription);
+        }
+
+        [Test]
+        public void VerifyIsAssignableFromGenericFixWithMessageAndArrayParamsInNonstandardOrder()
+        {
+            var code = TestUtility.WrapInTestMethod(@"
+            var actual = 42;
+
+            ↓ClassicAssert.IsAssignableFrom<int>(args: new[] { ""first"", ""second"" },  message: ""{0}, {1}"", actual: actual);");
+            var fixedCode = TestUtility.WrapInTestMethod(@"
+            var actual = 42;
+
+            Assert.That(actual, Is.AssignableFrom<int>(), $""{""first""}, {""second""}"");");
+            RoslynAssert.CodeFix(analyzer, fix, expectedDiagnostic, code, fixedCode, fixTitle: ClassicModelAssertUsageCodeFix.TransformToConstraintModelDescription);
+        }
+
+        [Test]
+        public void CodeFixMaintainsReasonableTriviaWithEndOfLineClosingParen()
+        {
+            var code = TestUtility.WrapMethodInClassNamespaceAndAddUsings(@"
+        public void TestMethod()
+        {
+            var expected = typeof(int);
+            var actual = 42;
+
+            ↓ClassicAssert.IsAssignableFrom(
+                expected,
+                actual,
+                ""message"");
+        }");
+            var fixedCode = TestUtility.WrapMethodInClassNamespaceAndAddUsings(@"
+        public void TestMethod()
+        {
+            var expected = typeof(int);
+            var actual = 42;
+
+            Assert.That(
+                actual,
+                Is.AssignableFrom(expected),
+                ""message"");
+        }");
+            RoslynAssert.CodeFix(analyzer, fix, expectedDiagnostic, code, fixedCode, fixTitle: ClassicModelAssertUsageCodeFix.TransformToConstraintModelDescription);
+        }
+
+        [Test]
+        public void CodeFixMaintainsReasonableTriviaWithNewLineClosingParen()
```

Comment:
I diffed `IsAssignableFromClassicModelAssertUsageCodeFixTests.cs` and `IsNotAssignableFromClassicModelAssertUsageCodeFixTests.cs` and in the other file this test is parametrized with `[Values] bool hasMessage`. Is there a reason that this one different?

Reply:
The two files were copied from `IsInstanceOf` and `IsNotInstanceOf` respectively.
Those have the same differences.
I have aligned all 4 now.